README.md

```diff
@@ -25,6 +25,10 @@ In general, these are the guidelines we consider when deciding what to pre-insta
 - Tools and versions will typically be removed 6 months after they are deprecated or have reached end-of-life.
 - If a tool can be installed during the build, we will evaluate how much time is saved
  and how much space is used by having the tool pre-installed.
+- If tool license is not MIT, Apache or GNU, the tool cannot be installed on image.

```

Comment:
> If tool license is not MIT, Apache or GNU, the tool cannot be installed on image

`If tool license it not MIT, GNU or Apache, we will have to check it with legal`
I think some tools with custom license may be installed

> Big tools can be pre-installed after our evaluation of the image space usage.

Looks like this point is already covered above
```
If a tool can be installed during the build, we will evaluate how much time is saved
 and how much space is used by having the tool pre-installed.
```

> Please, before creating a PR with tool installation, wait for confirmation that this tool can be installed on image.

I think we should add Note below the list.

```
**Note:** For new tools, please create an issue and get approval that we want to add this tool to the image before creating the pull-request.
```

Reply:
Thanks, fixed.